components/bucketItem/BucketItemList.tsx

```diff
@@ -1,22 +1,28 @@
 import React from 'react';
 import { useRouter } from 'next/router';
-import { useFetchBucketItems } from '@/hooks/useFetchBucketItems';
+import { useFetchBucketItems } from '@/hooks/bucketlist/useFetchBucketItems';
 import BucketItem from '@/components/bucketItem/BucketItem';
 import { Button, Input, InputGroup, InputRightElement } from '@chakra-ui/react';
+import { useMutateBucketItems } from '@/hooks/bucketlist/useMutateBucketItems';
 
 const BucketItemList = () => {
   const router = useRouter();
   const { folderId } = router.query;
 
   const { data, status } = useFetchBucketItems(Number(folderId));
+  const { createBucketItemMutation } = useMutateBucketItems();
+  const { mutate: createBucket } = createBucketItemMutation;
```

Comment:
(minor)
```suggestion
  const {mutate: createBucket } = useMutateBucketItems();
```
도 괜찮을 것 같습니다!

Reply:
앗 ㅠㅠ맞아요,, ,제가 Mutation에다가 다 몰아놔서 이렇게됐네요 ㅠㅠ create update delete mutation을 다 분리해놔야하나싶네요